contracts/nft/enumerable-nft.ral

```diff
@@ -0,0 +1,17 @@
+import "std/nft_interface"
+
+// Only 1 token is supported
+Contract EnumerableNFT(
+    collectionId: ByteVec,
```

Comment:
[Nit]: We can use the `collection: NFTPreDesignedCollection`, and we can get the `collectionId` by `tokenId!(collection)`

Reply:
Is there an easy way to fix [EnumerableNFT.encodeImmFields](https://github.com/alephium/alephium-nft/blob/nft-standard/contracts/nft/nft-pre-designed-collection.ral#L14) if we update the collectionId to collection?